src/main/java/uk/ac/ebi/ampt2d/accession/AccessionRepository.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.ampt2d.accession;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Accession repository
+ *
+ * @param <T> Object class
+ */
+public interface AccessionRepository<T> {
+    /**
+     * Given a list of objects, it returns the accessions for those objects that are stored in the repository
```

Comment:
For those not in the repository, is the key present in the `Map` and mapped to a null value, or it the key not even present? Please clarify in the documentation.